ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -30,9 +30,11 @@
     private static final By START_VISIT = By.id("org.openmrs.module.coreapps.createVisit");
     private static final By CONFIRM = By.cssSelector("#quick-visit-creation-dialog .confirm");
     private static final By STARTED_AT = By.className("active-visit-started-at-message");
-    private static final By VISIT_NOTE = By.linkText("Visit Note");
-    private static final By PATIENT_CAPTURE_VITALS = By.linkText("Capture Vitals");
+    private static final By VISIT_NOTE = By.id("referenceapplication.realTime.simpleVisitNote");
+    private static final By PATIENT_CAPTURE_VITALS = By.id("referenceapplication.realTime.vitals");
+    private static final By APPOINTMENT_EDIT = By.cssSelector("div.info-section:nth-child(4) > div > a:nth-child(3)");
```

Comment:
We would rather get better element references than retrieving the element using `div` and nth-child(`x`) . 
Also greeting a better element name would be great than APPOINTMENT_EDIT which seemingly sounds not clear !

Reply:
> Also greeting a better element name would be great than APPOINTMENT_EDIT which seemingly sounds not clear !

Which name will be clear to us

Reply:
If its a link then `APPOINTMENT_LINK` sounds clearer than `APPOINTMENT_EDIT`

Worth looking into the first review [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/521#discussion_r694666354)